usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -595,6 +595,7 @@ def load_file_c(submission_attributes, db_cursor, award_financial_frame):
     total_rows = award_financial_frame.shape[0]
     start_time = datetime.now()
     awards_touched = []
+    awards = Award.objects  # this stops the property lookup each iteration, saving 3+ seconds every 100 rows!
```

Comment:
Huh, I guess I vaguely realized that was happening, but it hadn't occurred to me that it could add up to measurable time.  :the-more-you-know: